alg/CATENets/requirements_dev.txt

```diff
@@ -0,0 +1,7 @@
+-r ./requirements.txt

```

Comment:
Inside this requirements file the library versions are specified exactly. I'd say if we are publishing this as a package, we should relax those requirements - perhaps within the specific major version of the libraries (or possibly within some minor versions, needs some research to check that API is compatible).

Reply:
I will update the versions.

Reply:
Thanks.